types/react-native/test/index.tsx

```diff
@@ -356,6 +360,25 @@ export class TouchableNativeFeedbackTest extends React.Component {
     }
 }
 
+// PressableTest
+export class PressableTest extends React.Component {
+    onPressButton = (e: GestureResponderEvent) => {
+        e.persist();
+        e.isPropagationStopped();
+        e.isDefaultPrevented();
+    };
+
+    render() {
+        return (
+            <Pressable onPress={this.onPressButton}>
```

Comment:
As a test for Pressable, can you include the style/children as a function that takes press state approach since that is unique?